### ISSUE TITLE
Task List: Some elements not accessible via keyboard

### ISSUE THREAD

**Issue by krallowski** [humhub/tasks#329]
In the list of tasks some elements are not part of the tab order and thus not reachable via keyboard.

### What steps will reproduce the problem?
1. Create some tasks
2. Go to the overview of tasks
3. Try to reach the following elements via keyboard

- Arrow key to expand / collapse the tasks
- Cross that appears on hover to reorder tasks
- clock icon and bubble icon to show the task details

### What is the expected result?
I can reach them

### What do you get instead?
I cannot reach them

### Additional info

| Q                | A
| ---------------- | ---
| HumHub version   | 1.18
| PHP version      | 
| Operating system |

> Note: Also provide the full error stacks from `Administration --> Information --> Logging` and errors from your browser console if possible.
> Note: Please use the [community](https://community.humhub.com) site for any questions on customizing HumHub.

**Comment by yurabakhtin** [comment]
@luke- PR https://github.com/humhub/tasks/pull/330
